client/src/components/CookieAlert.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+import CloseX from "../components/Icon/close-button";
+import { theme } from "../constants/theme";
+
+const { colors } = theme;
+
+const CookieContainer = styled.div`
+  alignitems: center;
+  background: ${colors.darkerGray};
+  color: ${colors.selago};
+  display: flex;
+  font-family: Poppins;
+  font-size: 1rem;
+  line-height: 1.5rem;
+  justify-content: center;
+  padding: 1.5rem 2rem;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  text-align: left;
+`;
+
+const CookieLink = styled.a`
+  color: ${colors.selago};
+  text-decoration: underline;
+`;
+
+const CloseButton = styled.button`
+  align-self: start;
+`;
+
+export default () => {
+  const [active, setActive] = useState();
+
+  const hideMessage = (event) => {
+    active ? setActive(false) : setActive(true);
+    localStorage.setItem("cookieMessage", "true");
```

Comment:
@robinv85 Just wanted to double check - is it fine to use localStorage for production? I remember reading a comment in the code base somewhere about localStorage not working well with production.

Reply:
Not sure what you mean with it not working, I first used local storage for some things like storing the answers to the wizards, email input etc. ... which was just a temporary solution because I didn't know what we were supposed to do with the data and the backend wasn't ready yet...
So that said, there is also a security concern for storing things like a JWT which is not relevant here
And as far as legal implications for the cookie law I'm also not sure if this applices to local storage, but I think it's probably just fine, but maybe we need to keep track of every data we store and where, for the legal team so they know to find everything

Reply:
Also if I think more about it, for anonymous users this probably works best
For a logged-in user we might even want to save it with their user data so if they log in on another browser or device they don't need to accept it again, but that's also something we should check with legal

Reply:
Ah I didn't mean local storage isn't working, I just wanted to take caution because there's a comment in `authActions.js`:

```
// Note: for production apps, both localstorage & cookies contain risks to store user & auth data
```
But it makes sense now that you mention auth in the picture. So this PR should be ok to merge.